### PR TITLE
Fix #552 - Allow stopClass for PropertyUtilsBean

### PR DESCRIPTION
**Pull request by Rapster**
Fix https://issues.apache.org/jira/projects/BEANUTILS/issues/BEANUTILS-552
- Add new method PropertyUtilsBean#getPropertyDescriptors(beanClass, stopClass)
- Add ctor for PropertyUtilsBean

### CODE REVIEW

**Comment by melloware** [comment]
@Rapster I have this fork of Commons BeanUtils2 so I could release on Maven Central since there has not been a release in 5 years if you want the submit the patch on my fork and I can push to Maven Central.

https://github.com/melloware/commons-beanutils2

**Comment by Rapster** [comment]
So what's the future of this repo then? I can see recent changes on this, but indeed no release so far...

**Comment by garydgregory** [comment]
> So what's the future of this repo then? I can see recent changes on this, but indeed no release so far...

The future is the same as for any Apache Commons component: We are volunteering our time and that is a limited resource. Someone needs to take the time and work on it. Since the next release is a major release, extra care must be taken that the API is what we really want it to be as it will be set in stone.

**Comment by Rapster** [comment]
Let me rephrase it as it's not what I meant, as a contributor myself I can only understand you're doing this on your free time, and appreciate very much the quality of this work.

As I just submitted a PR, and @melloware mentionned his own fork but I see you've been comitting things recently, understand that I was a bit confused...

I understand now a major release is on his way, and this repo is the one to look at. Look forward to see the result.

Keep up the good work 😉

**Comment by melloware** [comment]
@Rapster the last release of BeanUtils was 2016.  There was no release in sight and @garydgregory is extremely busy supporting all the Apache libs.  So there is no timeframe on a release.  My company got tired of waiting since it allowed me to remove Commons Collections 3 dependency and add 5 years worth of enhancments that is why I released it to Maven Central.
